server/ollamacfg/config.go

```diff
@@ -0,0 +1,167 @@
+package ollamacfg
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+type Config struct {
```

Comment:
At the risk of another package mutating the values, could we make these top level variables?

```
var Allow Origins []string
```

Coupled with the `init()` idea below they could be set before the main program runs

Reply:
OK.  Let me do some refactoring to try to retain a unit friendly [re]initialization flow, but remove the mutex and do package globals.